### PR TITLE
implement all my basic routes and testing routes

### PR DESCRIPTION
**Pull request by eugene-chu**
All my routes will get a random set of contiguous data and return back to the client.
Any where between 1-10M in both places and things-todo database can be selected, and a 'random' amount each will be returned (3-12 objects for places and 5-20 objects for things todo)

### CODE REVIEW

**Comment by eugene-chu** [comment]
Also implement the testing routes that grabs the last 12 objects from the places table and the last 20 objects from the todos table and return it back to the client

File: server/newrelic.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDCServer'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'f504babc10a4424c2f380c87ce023c61b54dNRAL',
```
Comment (Garrub) [inline server/newrelic.js]: I think you'll want to get this file removed from the git cache and then gitignore it, as your New Relic API Key exists here in plain text. Alternatively, abstract this key away into an environment variable that is then used by this file

**Review by Garrub — CHANGES_REQUESTED** [review]
Overall looks good, with one small issue in having your New Relic key in a git tracked file. I'm requesting changes as opposed to merging to give you an opportunity to prevent your API key from showing up in the master branch

File: server/newrelic.js (hunk 1 above)
Comment (eugene-chu) [reply server/newrelic.js]: Good catch, I will remove and amend this issue

**Review by Garrub — APPROVED** [review]
Looks good now!

**Pull request merged**